### ISSUE TITLE
Support wallrun and walljump

### ISSUE THREAD

**Issue by idanarye** [idanarye/bevy-tnua#12]

**Comment by Microwonk** [comment]
Is this a feature you still plan on doing? Currently trying to implement a Walljump TnuaAction for 2d so I thought this issue might be relevant.

**Comment by idanarye** [comment]
Yes, it's still planned, but it's not don't hold your breath - I can't promise any timeline. Before I get to this, I want to focus a bit on some more basic functionality (#10, #11 and #30)
